libdnf/rpm/rpm_signature.cpp

```diff
@@ -88,12 +88,14 @@ KeyInfo::KeyInfo(
     const std::string & key_id,
     const std::vector<std::string> & user_ids,
     const std::string & fingerprint,
+    const long int timestamp,
```

Comment:
`timestamp` is passed by value. The `const` keyword is not appropriate here.

Reply:
Yes, thanks!